zipline/sources/benchmark_source.py

```diff
@@ -107,8 +175,11 @@ def _validate_benchmark(self, benchmark_asset):
                 end_dt=benchmark_asset.end_date
             )
 
-    def _initialize_precalculated_series(self, asset, trading_calendar,
-                                         trading_days, data_portal):
+    def _initialize_precalculated_series(self,
+                                         asset,
+                                         trading_calendar,
+                                         trading_days,
+                                         data_portal):
```

Comment:
This is probably in progress as you write more tests, but it looks like the docstring needs the new return type and if `self.emission_rate == "minute"` the return type is only a Series

Reply:
good catch